ui/src/telegrafs/actions/index.ts

```diff
@@ -156,3 +159,15 @@ export const removeTelelgrafLabelsAsync = (
     dispatch(removeTelelgrafLabelFailed())
   }
 }
+
+export const downloadTelegrafConfig = (
+  telegrafConfigID: string,
+  telegrafConfigName: string
+) => async (dispatch): Promise<void> => {
+  try {
+    const config = await client.telegrafConfigs.getTOML(telegrafConfigID)
```

Comment:
since this config needs to be displayed by the overlay, could the same config be passed in so we dont have to make this request twice?